lib/rubygems/gemcutter_utilities.rb

```diff
@@ -22,7 +22,7 @@ module Gem::GemcutterUtilities
   def add_key_option
     add_option('-k', '--key KEYNAME', Symbol,
                'Use the given API key',
-               'from ~/.gem/credentials') do |value,options|
+               'from ~/.config/gem/credentials') do |value,options|
```

Comment:
Does this mean users will need to manually move `~/.gem/credentials` to `~/.config/gem/credentials/` or login again?

Reply:
No, When the old credential location was available, RubyGems picked it. 

see. https://github.com/ruby/ruby/pull/2174/files#diff-f1b45434a730b5cfbe4bac4f88479c7fR280

Reply:
Ahh, ok. Thx for clarification. I wasn't sure.